test_utils/__init__.py

```diff
@@ -17,3 +19,20 @@ def patch_site_configs(values):
     Use either as `@patch_site_configs(...)` or `with patch_site_configs(...):`.
     """
     return patch.dict('openedx.core.djangoapps.site_configuration.helpers.MOCK_SITE_CONFIG_VALUES', values)
+
+
+def create_standard_test_users():
```

Comment:
Looks like you could make this a fixture in `conftest.py` : https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions